static/sass/elements/chromedash-roadmap-css.js

```diff
@@ -0,0 +1,15 @@
+import { css } from "lit";
+import { SHARED_STYLES } from "../shared-css.js";
+import { LAYOUT_CSS } from "../_layout-css.js";
+
+export const ROADMAP_CSS = [
+  SHARED_STYLES,
+  LAYOUT_CSS,
+  css`
+    :host {
+      display: inline-flex;
+      padding: 0 0em $content-padding * 5;
```

Comment:
These are SCCS variables that would need to be replaced with JS constants or CSS variables.   There is no multiplication in CSS expressions, so you would need a constant that is 5 times the $content-padding value.  Or, you could just do the multiplication yourself and use 80px here.

Reply:
Good catch!  I noticed in other places.  But the math would need some kind of expression evaluation.  I wonder if something like this might be possible css`... ${cssvar`--content-padding` * 5} ...`, assuming cssvar could be defined to do the right thing.

Reply:
However, the next line with margin-right: $content-padding * -1  is more likely to be intended as relative to content-padding.  Maybe we should define more things similar to --content-padding-half, like --content-padding-negative, and --content-padding-huge.